src/components/Actions/utils.js

```diff
@@ -245,3 +245,17 @@ export const removeQualification = async ({ client, files, showAlert, t }) => {
     })
   }
 }
+
+/**
+ * serializeFilesForNavigateState - Serialize files for navigate state
+ *
+ * It is not possible to pass everything we want as a state in react-router-dom navigate function.
+ * For example in `navigate('/route', { state: data })`, it is not possible to have functions in data.
+ * Here we remove the `bills` attribute which may contain functions.
+ *
+ * @param {object} options
+ * @param {import('cozy-client/types/types').IOCozyFile[]} options.files  One or more files to serialize for navigate state
+ */
+export const serializeFilesForLocationState = ({ files }) => {
+  return files.map(file => ({ ...file, bills: undefined }))
```

Comment:
Je ne sais pas si il ne serait pas mieux de déshydraté les fichiers.
Pour peu que demain les fichiers soient hydratés avec d'autres relations, on retombera dans le problème.

Qu'en penses-tu ?

Reply:
There is a `dehydrate` method I think in CC.

Also, I can understand how easy it is to use the state mechanism of RR, but you can also use an outlet, no? With that, no need to `dehydrate` 

Or you should find a way to make relationship serializable. Only removing one relation like that feels a bit flaky

Reply:
That was indeed a very bad fix... Using `dehydrate` from cozy-client is as easy and a lot stronger.

Even if I agree that the state mechanism of RR is not the best tool here, I do not want to refactor it now.